pages/lists/[listId]/contributors.tsx

```diff
@@ -54,17 +57,24 @@ interface ContributorListPageProps {
 const ContributorsListPage = ({ list, initialData, isError }: ContributorListPageProps) => {
   // create useIsOwner(list?.user_id, userId) once we're ready to implement this.
   const isOwner = false;
+  const router = useRouter();
+  const { range } = router.query;
+
+  useEffect(() => {
+    if (!range) {
+      setQueryParams({ range: "30" });
+    }
+  }, [range]);
```

Comment:
Same as above. No need for the setting the range query param.

```suggestion
  const { range = "30" } = router.query;
```